bin/flutter

```diff
@@ -16,27 +16,31 @@ set -e
 
 unset CDPATH
 
-function follow_links() {
-  cd -P "${1%/*}"
-  local file="$PWD/${1##*/}"
+# The following is enclosed in a subshell to avoid changing the cwd of the
+# caller.
+function follow_links() (
```

Comment:
substituting `{` -> `(` means the function body runs in a subshell? I've never seen this before...

Reply:
Yes, that's what it means.  That's why I added the comment too.